utilities/rpp-performancetests/HIP_NEW/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ if (hip_FOUND)
     add_executable(Tensor_hip_pln3 Tensor_hip_pln3.cpp)
     # add_executable(Single_hip Single_hip.cpp)
     add_executable(uniqueFunctionalities_hip uniqueFunctionalities_hip.cpp)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fopenmp -DGPU_SUPPORT=1 -DHIP_COMPILE=1 -std=c++11")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fopenmp -DGPU_SUPPORT=1 -DRPP_BACKEND_HIP=1 -std=c++11")
     target_link_libraries(BatchPD_hip_pkd3 ${OpenCV_LIBS} -lamd_rpp ${HIP_LIBRARIES} pthread boost_filesystem boost_system hip::host)
```

Comment:
link libraries for the hip is hip::host which might make it run on host (please double check). I think it should be hip::device